x/ecocredit/core/types_class_issuer.go

```diff
@@ -0,0 +1,19 @@
+package core
```

Comment:
No changes here. Moved from `core/genesis.go` (and what is now `genesis/genesis.go`).